lib/WooCommerce/TransactionalEmails/Renderer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace MailPoet\WooCommerce\TransactionalEmails;
+
+use csstidy;
+use MailPoet\Models\Newsletter;
+use MailPoet\Newsletter\Renderer\Preprocessor;
+use MailPoet\Newsletter\Renderer\Renderer as NewsletterRenderer;
+
+class Renderer {
+  const CONTENT_CONTAINER_ID = 'mailpoet_woocommerce_container';
+
+  /** @var string */
+  private $html_before_content;
+
+  /** @var string */
+  private $html_after_content;
+
+  function __construct() {
+    $this->html_before_content = '';
+    $this->html_after_content = '';
+  }
+
+  public function render(Newsletter $newsletter) {
+    $renderer = new NewsletterRenderer($newsletter, true);
+    $html = explode(Preprocessor::WC_CONTENT_PLACEHOLDER, $renderer->render('html'));
+    $this->html_before_content = $html[0];
+    $this->html_after_content = $html[1];
+  }
+
+  public function getHTMLBeforeContent($heading_text) {
```

Comment:
I must admit this class looks quite odd to me. All methods are public. It is a one-shot class... You can instantiate it once, call render (which doesn't return anything) and after that using getters pull pieces of rendered code. And the `prefixCss` is here for an unknown reason... 

Could this be done as a reusable service?